### PR TITLE
[NOJIRA] Initial release

### PR DESCRIPTION
**Pull request by imrekel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
...
+[![Step changelog](https://shields.io/github.com/bitrise-steplib/bitrise-step-restore-xcodebuild-cache?include_prereleases&label=changelog&color=blueviolet)](https://github.com/bitrise-steplib/bitrise-step-restore-xcodebuild-cache/releases)
+
+Restores the DerivedData folder and the related metadata from Bitrise Build Cache to speed up subsequent builds.
+
+<details>
+<summary>Description</summary>
+
+This step is the pair of the [save-xcodebuild-cache step](https://github.com/bitrise-steplib/bitrise-step-save-xcodebuild-cache) and can be used to restore xcodebuild cache archives from the Bitrise Build Cache. 
+
+As xcodebuild only reuses DerivedData if the input files' modification time is the same, the step also restores the modification time of the input files (project files including source code files) from a metadata file stored along the
```
Comment (zsolt-marta-bitrise) [inline README.md:12]: > use the same build settings/settingsas

What did you want to write here?

Comment (zsolt-marta-bitrise) [reply README.md:12]: Also pls check the newline, seems placed wrongly

File: README.md [hunk 2]
```diff
@@ -1 +1,51 @@
...
+
+## ⚙️ Configuration
+
+<details>
+<summary>Inputs</summary>
+
+| Key | Description | Flags | Default |
+| --- | --- | --- | --- |
+| `project_root_path` | Path to the root folder of the project to be built | required | `.` |
+| `cache_key` | Path to the root folder of the project to be built |  |  |
```
Comment (zsolt-marta-bitrise) [inline README.md]:
```suggestion
| `cache_key` | The key used to store the cache archive |  |  |
```

File: e2e/bitrise.yml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            set -ex
+            set -o pipefail
+
+            defaults write com.apple.dt.XCBuild IgnoreFileSystemDeviceInodeChanges -bool YES
+
+            xcodebuild build -scheme Bitrise\ TODOs\ Sample CODE_SIGN_IDENTITY= CODE_SIGNING_REQUIRED=NO
+    - git::https://github.com/bitrise-steplib/bitrise-step-save-xcodebuild-cache.git@prepare-release:
```
Comment (zsolt-marta-bitrise) [inline e2e/bitrise.yml:82]: Is this the final branch of the e2e test?

File: README.md (hunk 1 above)
Comment (lpusok) [reply README.md:12]: Please edit step.yml for description, as README is autogenerted.

Comment (imrekel) [reply README.md:12]: oh, good to know :D

File: e2e/bitrise.yml (hunk 3 above)
Comment (imrekel) [reply e2e/bitrise.yml:82]: we should end up using the `main` branch eventually

**Review by zsolt-marta-bitrise — APPROVED** [review]

**Pull request merged**